config/locales/whats_new.yml

```diff
@@ -0,0 +1,67 @@
+en:
+  whats_new:
+    show_banner: false
+    title: What’s new in Manuals Publisher
+    summary: |
+      Summary of updates
+    last_updated: Last updated 3 Jul 2023
+    our_roadmap:
+      heading: Our roadmap
+      body_govspeak: |
+        For now here’s the list of the features we’re working on, with a caveat that we may or may not get the time to deliver them all:
```

Comment:
It's a bit odd to tell users we might not add features - they don't have the context of a team working on it for a short period of time, but as per comment above this section is hopefully not needed.

Reply:
the team does have a replacement section here reworded in consultation with Mike W